### PR TITLE
AHTI-229 | Implement index page

### PR DESCRIPTION
**Pull request by ahie**

### CODE REVIEW

File: src/common/ui-components/ListView/ListView.module.scss [hunk 1]
```diff
@@ -2,6 +2,11 @@
   padding: 1rem;
 }
 
+.divider {
+  height: 1px;
+  background: black;
```
Comment (marharyta) [inline src/common/ui-components/ListView/ListView.module.scss]: I understand this is work in progress, but a quick note: magic number

File: src/styles/partials/_mixins.scss [hunk 2]
```diff
@@ -26,3 +26,16 @@
   border: none;
   background: none;
 }
+
+@mixin helsinki-logo($_color) {
```
Comment (marharyta) [inline src/styles/partials/_mixins.scss]: should be in its own file, maybe a variable.

File: src/styles/index.scss [hunk 3]
```diff
@@ -51,7 +51,7 @@ body {
   font-family: $font-family;
 
   #root {
-    width: 474px;
+    width: 375px;
```
Comment (marharyta) [inline src/styles/index.scss]: magic number

File: src/styles/partials/_mixins.scss (hunk 2 above)
Comment (ahie) [reply src/styles/partials/_mixins.scss]: Not sure where you want to move this.

File: src/common/ui-components/ListView/ListView.module.scss (hunk 1 above)
Comment (ahie) [reply src/common/ui-components/ListView/ListView.module.scss]: Fixed.

File: src/styles/index.scss (hunk 3 above)
Comment (ahie) [reply src/styles/index.scss]: Fixed.

**Pull request merged**